ACMILabs/acmi-api
# #100 Add audio API

Resolves #100

Let's add an `audio` API to power future experiments.

### Acceptance Criteria
- [x] Add `audio` API

### Relevant design files
* None

### Testing instructions
1. Rebuild your environment: `make build`
1. Reindex your local search (see below)
1. Get a list of audio: http://localhost:8081/audio/
1. Get a specific audio item: http://localhost:8081/audio/15/
1. Get an audio item by label IDs: localhost:8081/audio/?labels=61958,47741
1. Try searching for an audio file: http://localhost:8081/search/?query=marshmallow&resource=audio
1. Try using the `raw` filter: http://localhost:8081/search/?query=marshmallow&resource=audio&raw=true

```python
$ docker exec -it api python3

from app.api import Search
search = Search()
s.update_index('audio')
```
